locale/es.po

```diff
@@ -473,9 +473,9 @@ msgstr "[a] abortar la compilación de todos los paquetes"
 msgid "[c] checksums skip"
 msgstr "[c] omisión de sumas de comprobación"
 
-#: pikaur/install_cli.py:463
+#: pikaur/install_cli.py:464
 msgid "[c] git checkout -- '*'"
-msgstr "[c] git checkout -- '*'"
+msgstr "[c] extracción git -- '*'"
```

Comment:
already after merging found this problem (and the same in `fr.po`)

(unless with es/fr locale you actually could ask `git` to make some extrazione for you)